### ISSUE TITLE
Failed installation, too

### ISSUE THREAD

**Issue by JingHuiTong** [paudetseis/Telewavesim#45]
Hi~ 
I'm glad that I can use Telewavesim to simulate SKS waveform. 
But I have an installation problem! 

My environment: python 3.7 (MacOS ver 11.2.3 for MacBook air M1) & installed gfortran
(also I tested in my Mac mini MacOS 10.15)

When I import telewavesim, I had error about:
`OSError: dlopen(/Users/tong/opt/anaconda3/envs/tws/lib/libncurses++.a, 6): no suitable image found.  Did find:
	/Users/tong/opt/anaconda3/envs/tws/lib/libncurses++.a: unknown file type, first eight bytes: 0x21 0x3C 0x61 0x72 0x63 0x68 0x3E 0x0A
	/Users/tong/opt/anaconda3/envs/tws/lib/libncurses++.a: unknown file type, first eight bytes: 0x21 0x3C 0x61 0x72 0x63 0x68 0x3E 0x0A`

Surely, I also had re-installed. 
The problem is urgent to me, thank you in advance.

**Comment by trichter** [comment]
Hi, can you try to install the development version with e.g.

```
pip install https://github.com/paudetseis/Telewavesim/archive/master.zip
```

and check your error persists?

**Comment by JingHuiTong** [comment]
Yes!!! the development version can successfully import. 
Thank you so much!

**Comment by trichter** [comment]
Great, that it worked. The fix is included in the latest release v0.2.1 (released 12 hours ago).
